controller/scripts/entrypoint-test.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 set -x
```

Comment:
could you remove all `set` lines in this file? IMO prints like `+ PYCODESTYLE=0` are just noise

Reply:
Done